src/python/director/startup.py

```diff
@@ -1,6 +1,8 @@
 # This script is executed in the main console namespace so
 # that all the variables defined here become console variables.
 
+from __future__ import division
```

Comment:
ok, i see in the docs that the import will only be valid for code in startup.py and won't extend to the console, since startup.py is run with an exec statement.

Would you mind removing the import?